core/meta/src/TClass.cxx

```diff
@@ -1329,6 +1329,8 @@ void TClass::Init(const char *name, Version_t cversion,
    if (!gROOT)
       ::Fatal("TClass::TClass", "ROOT system not initialized");
 
+   assert(name != 0 && strlen(name) != 0 || 0 == "TClass::Init, the name parameter is invalid (null or empty)");
```

Comment:
Can we make this an `Error()`? Users might happen to call this as `TClass::GetClass("")`; we should be "nice" and return an error then instead of going on in release or terminating in debug. And the cheaper version of this check is `if (!name || !name[0])` (doesn't need a `strlen`).

Reply:
Done. Thanks for the comment